### PR TITLE
Prepare for 3.6.1 release

### PR DESCRIPTION
**Pull request by ChrisMaddock**
Had a spare half hour, so thought I'd get ahead of the game with this. 

I think we should still get #75 and #85 into this release - can merge those in and update the release notes once they're done.

Have I missed anything? 😄

### CODE REVIEW

**Comment by rprouse** [comment]
Looks good so far. I don't see anything you've missed, but I haven't released this since 3.0 😄

File: CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+### NUnit Xamarin Runner 3.6.0 - January 26, 2017
```
Comment (rprouse) [inline CHANGES.md]: 3.6.0 was never released, so this should be changed to the current release 3.6.1 and recently fixed issues added.

File: build.cake [hunk 2]
```diff
@@ -12,7 +12,7 @@ var configuration = Argument("configuration", "Release");
 var isLocal = BuildSystem.IsLocalBuild;
 var isRunningOnAppVeyor = AppVeyor.IsRunningOnAppVeyor;
 
-var version = "3.0.1";
+var version = "3.6.0";
```
Comment (rprouse) [inline build.cake]: I guess you are going to pull in recent changes which will update this to 3.6.1?

File: src/runner/nunit.runner/Properties/AssemblyCommon.cs [hunk 3]
```diff
@@ -26,12 +26,12 @@
...
 [assembly: AssemblyCompany("NUnit Software")]
 [assembly: AssemblyProduct("NUnit 3 Xamarin Runner")]
-[assembly: AssemblyCopyright("Copyright (C) 2015 Charlie Poole")]
+[assembly: AssemblyCopyright("Copyright (C) 2017 Charlie Poole")]
 [assembly: AssemblyTrademark("NUnit is a trademark of NUnit Software")]
 [assembly: AssemblyCulture("")]
 
-[assembly: AssemblyVersion("3.0.*")]
-[assembly: AssemblyFileVersion("3.0.0.0")]
+[assembly: AssemblyVersion("3.6.*")]
```
Comment (rprouse) [inline src/runner/nunit.runner/Properties/AssemblyCommon.cs]: Should we change this to an explicit version rather than the *?

**Comment by ChrisMaddock** [comment]
Thanks for these Rob - I was planning to come back to this PR and update tonight. 🙂

The things I'm currently aware of are:
- Update version numbers
- Update Changelog with version/released date/new issues
- Update year on licence.txt
- Make draft release

The nuspec file was done in #85, this branch just needs rebasing. Anything else you can think of?

File: src/runner/nunit.runner/Properties/AssemblyCommon.cs (hunk 3 above)
Comment (ChrisMaddock) [inline src/runner/nunit.runner/Properties/AssemblyCommon.cs]: Seems sensible to me. I wasn't sure why they were *'s - so I just left them as they were!

**Comment by CharliePoole** [comment]
If, as @rprouse says, we never released 3.6.0, why is this 3.6.1? That seems potentially confusing.

**Comment by rprouse** [comment]
@ChrisMaddock that looks like a good list.

@CharliePoole it is going to be 3.6.1 because it is like NUnitLite and references the 3.6.1 framework so we thought they should match to prevent confusion.

**Comment by CharliePoole** [comment]
I don't think we should use *s in any of our versions.

**Pull request closed without merging**